web_external/js/views/body/StudyView.js

```diff
@@ -0,0 +1,26 @@
+isic.views.StudyView = isic.View.extend({
+    initialize: function (settings) {
+        girder.cancelRestRequests('fetch');
```

Comment:
@msmolens Does this cancel all requests globally? If so, what if we're loading 2 StudyViews concurrently?